src/Fieldset.php

```diff
@@ -391,7 +391,7 @@ public function populateValues($data)
         }
 
         foreach ($this->iterator as $name => $elementOrFieldset) {
-            $valueExists = array_key_exists($name, $data);
+            $valueExists = isset($data[$name]);
```

Comment:
`isset` is not the same as `array_key_exists` in case `$data[$name] === null`.

It is fine in line 397 because we expect there array or traversable
It is fine in line 403 because we skip null value there anyway

It can be an issue in line 415, when we accept null.